api/src/main/java/org/loomdev/api/item/property/ItemProperty.java

```diff
@@ -85,7 +85,7 @@
      * Check if the {@link ItemProperty} can be applied to an {@link ItemStack}.
      *
      * @param itemStack The {@link ItemStack} to check the {@link ItemProperty} against.
-     * @return True if the {@link ItemProperty van be applied}.
+     * @return {@code true} if the {@link ItemProperty van be applied}.
```

Comment:
should be `{@code true} if the {@link ItemProperty}  can be applied.` a spelling and format error on my part.